### ISSUE TITLE
Android app doesn't connect to server

### ISSUE THREAD

**Issue by starlingfire** [gotify/android#42]
Hi, apologies if I'm just doing something wrong, but I can't seem to get my Android client to sync up with the server. I have it running on a non-standard port (8118), and am able to connect to the server with my username and password. I can also see updates in the server log when I pull down to refresh on Android.
```
[GIN] 2019/01/11 - 17:00:02 | 200 |     554.496µs |     192.168.1.1 | GET      /stream?token=XXXXXXXXXXXX
[GIN] 2019/01/11 - 17:00:02 | 200 |     799.695µs |     192.168.1.1 | GET      /application
[GIN] 2019/01/11 - 17:00:02 | 200 |     868.698µs |     192.168.1.1 | GET      /message?limit=100&since=0
```
However, there are no messages. Through the terminal I can create messages, and I see them in the server web UI, but nothing on my phone. 

Also, even though I have logged in to the server from Android, the device does not appear in the list of clients.

Server log of starting gotify and adding my Android client:
```
$ sudo ./gotify-linux-amd64
Starting Gotify version 1.2.1@2018-12-12-21:08:19
Started Listening on port 8118
[GIN] 2019/01/11 - 17:06:04 | 200 |     210.118µs |     192.168.1.1 | GET      /version 
[GIN] 2019/01/11 - 17:06:15 | 200 |   52.376055ms |     192.168.1.1 | GET      /current/user
[GIN] 2019/01/11 - 17:06:21 | 200 |   62.132179ms |     192.168.1.1 | POST     /client
[GIN] 2019/01/11 - 17:06:21 | 200 |     661.908µs |     192.168.1.1 | GET      /current/user
[GIN] 2019/01/11 - 17:06:21 | 200 |     102.729µs |     192.168.1.1 | GET      /version
[GIN] 2019/01/11 - 17:06:21 | 200 |     917.238µs |     192.168.1.1 | GET      /message?limit=1&since=0
[GIN] 2019/01/11 - 17:06:21 | 200 |      898.51µs |     192.168.1.1 | GET      /stream?token=C8VbjR_XXXXX
[GIN] 2019/01/11 - 17:06:21 | 200 |     777.636µs |     192.168.1.1 | GET      /application
[GIN] 2019/01/11 - 17:06:21 | 200 |    2.197495ms |     192.168.1.1 | GET      /message?limit=100&since=0
[GIN] 2019/01/11 - 17:06:36 | 200 |      153.26µs |       127.0.0.1 | GET      /
[GIN] 2019/01/11 - 17:06:36 | 200 |     516.369µs |       127.0.0.1 | GET      /current/user
[GIN] 2019/01/11 - 17:06:36 | 200 |     132.877µs |       127.0.0.1 | GET      /static/css/main.c1fdf91b.css
[GIN] 2019/01/11 - 17:06:36 | 200 |     263.933µs |       127.0.0.1 | GET      /static/js/main.d0fe35f1.js
[GIN] 2019/01/11 - 17:06:36 | 200 |      223.44µs |       127.0.0.1 | GET      /current/user
[GIN] 2019/01/11 - 17:06:36 | 200 |      16.957µs |       127.0.0.1 | GET      /version
[GIN] 2019/01/11 - 17:06:36 | 200 |      71.387µs |       127.0.0.1 | GET      /service-worker.js
[GIN] 2019/01/11 - 17:06:36 | 200 |     349.347µs |       127.0.0.1 | GET      /application
[GIN] 2019/01/11 - 17:06:36 | 200 |     278.844µs |       127.0.0.1 | GET      /message?since=0
[GIN] 2019/01/11 - 17:06:36 | 200 |      80.306µs |       127.0.0.1 | GET      /static/defaultapp.png
[GIN] 2019/01/11 - 17:06:38 | 200 |     256.609µs |       127.0.0.1 | GET      /client
[GIN] 2019/01/11 - 17:06:41 | 403 |     555.275µs |       127.0.0.1 | GET      /stream?token=C8VbjR_YYYYYYYYY
[GIN] 2019/01/11 - 17:06:41 | 200 |     548.888µs |       127.0.0.1 | GET      /current/user
```

**Comment by starlingfire** [comment]
Okay fair warning, I did disclaimer that I might be doing something wrong, and it turns out that I was logged in as the admin user in my web browser, but as a different user on my android client.

I had assumed the admin user could see all client tokens, but I guess each user can only see their own? When I logged in as the second user in my browser, I saw all the different clients I created while troubleshooting my phone.
